src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -2685,6 +2685,14 @@ Please specify one of them using the --source option to proceed.</value>
     <value>Uri not well formed: {0}</value>
     <comment>{Locked="{0}"} Error message displayed when the provided uri is not well formed. {0} is a placeholder replaced by the provided uri.</comment>
   </data>
+  <data name="UriBlockedBySmartScreen" xml:space="preserve">
+    <value>This operation was blocked as unsafe by Microsoft Defender SmartScreen.</value>

```

Comment:
Adding @RDMacLachlan to help with the displayed messages (I copied the messages from App Installer with some modifications)